src/cqrs/error/LockViolationError.ts

```diff
@@ -0,0 +1,10 @@
+import { ReturnToCallerError } from './ReturnToCallerError';
+
+export class LockViolationError extends ReturnToCallerError {
```

Comment:
Why is this a ReturnToCallerError?

I'm not too sure that this should be such an error